pyopensprinkler/__init__.py

```diff
@@ -369,6 +369,26 @@ def last_run_end_time(self):
         """Retrieve last run end time"""
         return self._get_variable("lrun")[3]
 
+    @property
+    def rssi(self):
+        """Retrieve RSSI"""
+        return self._get_variable("RSSI")
+
+    @property
+    def latitude(self):
+        """Retrieve latitude"""
+        return float(self._get_variable("loc").split(",")[0].strip())
```

Comment:
Will `loc` always be set? Might need to check for `None` or empty string

Reply:
K, I'll code it up.

Reply:
Done.